pkg/engine/hybrid/hybrid.go

```diff
@@ -94,11 +94,6 @@ func New(options *types.CrawlerOptions) (*Crawler, error) {
 
 // Close closes the crawler process
 func (c *Crawler) Close() error {
```

Comment:
After some observation, I believe the `browser` should be closed. We can call [`browser.MustClose()`](https://pkg.go.dev/github.com/go-rod/rod#Browser.MustClose) as an alternative, regardless of the condition.

```suggestion
func (c *Crawler) Close() error {
	c.browser.MustClose()
```

Reply:
I've tried using `browser.MustClose`, but it doesn't force close the visiable browser with `Ctrl-c` for me. Does it work for you?

![Screenshot_20240730_090031](https://github.com/user-attachments/assets/46335990-20ff-4089-a1f4-51d8d592a596)

As shown in the screenshot, I have already added `browser.MustClose` and recompiled.

Reply:
> Does it work for you?

Yes, it works.

Reply:
**UPDATE**: after I've tried loads of URLs and doing interrupts, the `Crawler.browser` doesn't actually close. It's really strange.

The initial solution you gave worked well, but I'm not so sure it's the most graceful way, considering that we're also calling `Crawler.killChromeProcesses()`.